generate_brick.py

```diff
@@ -691,7 +691,7 @@ def add_definitions():
 res = G.query(
     """SELECT ?quantity ?qudtquant WHERE {
                 ?quantity rdf:type brick:Quantity .
-                ?quantity brick:hasQUDTReference ?qudtquant
+                ?quantity skos:broader*/brick:hasQUDTReference ?qudtquant
```

Comment:
Assuming narrower concepts are analogous to subclasses, broader concepts should encompass narrower concepts, but not the other way around.

I propose to change this to `skos:narrower` and change `qk:CelsiusTemperature` to `qk:ThermodynamicTemperature` if we want to keep `qk:ThermodynamicTemperature` instead of `qk:Temperature`.

Reply:
For example if I read the query, I don't expect to pull all the concepts other than `CelsiusTemperature` when I ask for `CeclsiusTemperature`. But if I query for `Temperature` (to avoid the `ThermodynamicsTempreature` issue), I expect `Celcius` to be included.

Reply:
After some investigation, I've found that QUDT is pretty consistent about having compatible units up and down the broader/narrower links. The real fix here is to ensure that `qk:Temperature` was used as the reference for `brick:Temperature`. Outside of that, I've noticed no change in the compiled model when we use the `skos:narrower` traversal. I think that we can avoid those hierarchy traversals for now, which should address the issues you are seeing in #305